tests/lib/screenshot-testing/support/page-renderer.js

```diff
@@ -121,14 +120,19 @@ PageRenderer.prototype._reset = function () {
 PageRenderer.prototype.waitFor = function (selectorOrTimeoutOrFunction) {
     console.log('Using page.waitFor is deprecated, please use one of this instead: waitForSelector, waitForFunction, waitForTimeout');
     if (typeof selectorOrTimeoutOrFunction === 'function') {
-        this.webpage.waitForFunction(selectorOrTimeoutOrFunction)
+        return this.webpage.waitForFunction(selectorOrTimeoutOrFunction)
     } else if (typeof selectorOrTimeoutOrFunction === 'number') {
-        this.webpage.waitForTimeout(selectorOrTimeoutOrFunction)
+        return this.webpage.waitForTimeout(selectorOrTimeoutOrFunction)
     } else if (typeof selectorOrTimeoutOrFunction === 'string') {
-        this.webpage.waitForSelector(selectorOrTimeoutOrFunction)
+        return this.webpage.waitForSelector(selectorOrTimeoutOrFunction)
     }
 }
 
+PageRenderer.prototype.type = async function (...args) {
+  await this.webpage.type(...args);
+  await this.waitForTimeout(100); // puppeteer types faster than vue can update the model state
```

Comment:
puppeteer types too fast for vue to update the model, so adding a timeout here fixed a bunch of UI tests.